sdk/messaging/azeventgrid/response_types.go

```diff
@@ -1,5 +1,3 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Seems like this shouldn't be removed?

Reply:
This is odd.

<img width="566" alt="image" src="https://github.com/Azure/azure-sdk-for-go/assets/51494936/f3bc6090-a0b7-4376-9aac-166f0810df75">

The diff is showing it's deleted but it's not in the branch...

Reply:
I'm looking at the actual file in the branch and it's not removed there. I think this might just be a PR cosmetics error. 

(if it gets removed after merge I'll add it back)